dotcom-rendering/src/web/components/Snap.stories.tsx

```diff
@@ -0,0 +1,20 @@
:500;font-size:16px;font-family:"Guardian Headline Bold","Guardian Headline Full","Guardian Egyptian Web",Georgia,serif;color:#fff;margin-top:10px;margin-bottom:10px;max-width:226px}@media (min-width:23.4375em){.us-morning-briefing-post-election-2020__blurb{margin-right:115px}}@media (min-width:30em){.us-morning-briefing-post-election-2020__blurb{margin-right:210px}}@media (min-width:41.25em){.us-morning-briefing-post-election-2020__blurb{margin-right:0;max-width:246px;font-size:17px;display:block}}@media (min-width:46.25em){.us-morning-briefing-post-election-2020__blurb{max-width:306px}}@media (min-width:61.25em){.us-morning-briefing-post-election-2020__blurb{font-size:20px;max-width:380px}}.us-morning-briefing-post-election-2020__component-button{display:inline-flex;flex-direction:row;background-color:#121212;border-radius:12px;text-decoration:none;cursor:pointer;height:36px;border-radius:26px}.us-morning-briefing-post-election-2020__component-button span{color:#fff;font-family:"Guardian Text Sans Web","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;font-weight:700;font-size:17px;padding:10px 2px 10px 20px;margin-top:-2px;text-decoration:none}.us-morning-briefing-post-election-2020__component-button svg{padding:7px;fill:#fff;padding-right:16px;transition:transform .3s ease}.us-morning-briefing-post-election-2020__component-button:hover{background-color:#767676;text-decoration:none}.us-morning-briefing-post-election-2020__component-button:hover span{text-decoration:none}.us-morning-briefing-post-election-2020__component-button:hover svg{transform:translateX(3px)}.us-morning-briefing-post-election-2020__component-button span{padding-top:6px;font-size:17px}.us-morning-briefing-post-election-2020__component-button:focus,.us-morning-briefing-post-election-2020__component-button:hover{background-color:#ffd622;text-decoration:none}.us-morning-briefing-post-election-2020__component-button:focus span,.us-morning-briefing-post-election-2020__component-button:hover span{color:#000;text-decoration:none}.us-morning-briefing-post-election-2020__component-button:focus svg,.us-morning-briefing-post-election-2020__component-button:hover svg{fill:#000}@media (min-width:41.25em){.us-morning-briefing-post-election-2020__component-button{margin-top:16px}}.us-morning-briefing-post-election-2020__roundel{top:4px;right:4px;position:absolute;top:3px;right:9px}@media (min-width:30em){.us-morning-briefing-post-election-2020__roundel{right:18px}}@media (min-width:30em){.us-morning-briefing-post-election-2020__roundel{right:10px;top:3px}}@media (min-width:46.25em){.us-morning-briefing-post-election-2020__roundel{right:11px;top:6px}}.interactive-atom{margin:0;overflow:hidden}.android .us-morning-briefing-post-election-2020,.ios .us-morning-briefing-post-election-2020{overflow:hidden;margin:0;padding:0}.android .us-morning-briefing-post-election-2020__component-button span,.ios .us-morning-briefing-post-election-2020__component-button span{letter-spacing:unset}',
```

Comment:
Is it typical of snaps to have `@font-face` declarations? [According to MDN, they will be discarded if inserted inside a css selector](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face#notes).

I think we should warn whenever we find such declaration, but this can be done in a subsequent PR.

Reply:
Unfortunately it is very common for snaps to have `@font-face`, I'm pretty sure that the emotion CSS parser is intelligent enough to not wrap `@font-face` in a selector